spec/rubocop/cop/style/end_block_spec.rb

```diff
@@ -9,4 +9,20 @@
       ^^^ Avoid the use of `END` blocks. Use `Kernel#at_exit` instead.
     RUBY
   end
+
+  it 'auto-corrects "END" with "at_exit"' do
+    new_source = autocorrect_source(<<~RUBY)
+      END { puts 'Goodbye!' }
+    RUBY
+    expect(new_source).to eq(<<~RUBY)
+      at_exit { puts 'Goodbye!' }
+    RUBY
+  end
+
+  it 'does not autocorrect other blocks' do
+    source = <<~RUBY
+      end_block { puts 'Goodbye!' }
+    RUBY
+    expect(autocorrect_source(source)).to eq(source)
+  end
```

Comment:
This is a small point. For example, the following would simplify the test code:

```diff
--- a/spec/rubocop/cop/style/end_block_spec.rb
+++ b/spec/rubocop/cop/style/end_block_spec.rb
@@ -3,26 +3,20 @@
 RSpec.describe RuboCop::Cop::Style::EndBlock do
   subject(:cop) { described_class.new }

-  it 'reports an offense for an END block' do
+  it 'registers an offense and corrects for an END block' do
     expect_offense(<<~RUBY)
       END { test }
       ^^^ Avoid the use of `END` blocks. Use `Kernel#at_exit` instead.
     RUBY
-  end

-  it 'auto-corrects "END" with "at_exit"' do
-    new_source = autocorrect_source(<<~RUBY)
-      END { puts 'Goodbye!' }
-    RUBY
-    expect(new_source).to eq(<<~RUBY)
-      at_exit { puts 'Goodbye!' }
+    expect_correction(<<~RUBY)
+      at_exit { test }
     RUBY
   end

-  it 'does not autocorrect other blocks' do
-    source = <<~RUBY
-      end_block { puts 'Goodbye!' }
+  it 'does not register offense when using other blocks' do
+    expect_no_offenses(<<~RUBY)
+      end_block { test }
     RUBY
-    expect(autocorrect_source(source)).to eq(source)
   end
 end
```

Reply:
Thanks. This is much better :)